crates/aptos/src/common/types.rs

```diff
@@ -1318,6 +1334,26 @@ impl TransactionOptions {
             gas_unit_price
         };
 
+        // Get sequence number for account
+        let (account, state) = get_account_with_state(&client, sender_address).await?;
```

Comment:
Does this send this request in parallel with the estimate gas price request?

Reply:
One important consideration here is that this increases the number of requests and thus necessary quota for sending transactions. Might not be a big problem with the CLI but definitely the case for SDK.

Reply:
Let's separate SDK and CLI discussions here.

This is the CLI, and it's okay to make 2 requests, it's already doing many requests per CLI command anyways.  There isn't much parallelism or optimization, but since it's user operated it's less sensitive to speed